extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/tunnel/AbstractGuacamoleTunnelService.java

```diff
@@ -714,4 +714,15 @@ public GuacamoleTunnel getGuacamoleTunnel(RemoteAuthenticatedUser user,
 
     }
 
+
+    public Integer getIdleConnections(Connection connection) {
```

Comment:
All functions must be documented with a proper JavaDoc comment. See the surrounding code for how this should be done.